docs/auth/README.md

```diff
@@ -0,0 +1,116 @@
+# Running Lighthouse on Authenticated Pages
+
+Standard usages of Lighthouse processes a page as a new user, with no previous session or storage data. This means that pages requiring authenticated access do not work without some additional setup. There are multiple ways to run Lighthouse on an authenticated page - this document focuses on the most flexible approach (Puppeteer), but mentions other approaches briefly at the end.
+
+If you just want to view the code, see `./example-lh-auth.js`.
+
+## The Example Site
+
+There are two pages on the site:
+
+1. `/` - the homepage
+2. `/dashboard`
+
+The homepage shows the login form, but only to users that are not signed in.
+
+The dashboard shows a secret to users that are logged in, but shows an error to users that are not.
+
+The server responds with different HTML for each of these pages and session states, so there are four different pages that must have passable Lighthouse SEO scores.
+
+You can run this server locally if you like:
```

Comment:
```suggestion
To run the server:
```

Reply:
Loses important meaning that this is optional.

Reply:
The way we (technical writers) usually handle that is:

```
(Optional) To run the server:
```

Keeps it concise while also keeping the information about optional-ity